python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -518,7 +526,10 @@ def run(self, sensorRef, templateIdList=None):
 
             if self.config.doMeasurement:
                 self.log.info("Running diaSource measurement")
-                self.measurement.run(diaSources, subtractedExposure)
+                if not self.config.doMeasurementNew:
+                    self.measurement.run(diaSources, subtractedExposure)
+                else:
+                    self.measurement.run(diaSources, subtractedExposure, exposure, templateExposure)
```

Comment:
Sorry, can you explain? What do you mean by two different tasks? The plan is to simply remove lines 529-531 (and the doDipoleFitting option) when we decide to remove the old measurement method. I should note that doDipoleFitting will actually run both tasks by default (due to its config).